sycl/plugins/esimd_cpu/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+
+# PI Esimd CPU  library
+# Create Shared library for libpi_esimd_cpu.so.
+
+include_directories("${sycl_inc_dir}")
+include_directories(${OpenCL_INCLUDE_DIR})
+include_directories(${LIBCMRT_INCLUDE})
```

Comment:
Is LIBCMRT_INCLUDE defined anywhere? Or this is deadcode?
Can you please remove if not used in the next PRs

Reply:
I have my own fully-working branch for this ESIMD_CPU support, and it looks like `LIBCMRT_INCLUDE` seems to be mistakenly copied from the working branch. It will be used in coming PR.

Reply:
I noticed that I removed the macro recently while cleaning my working branch. It will be removed in next PR.